CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # UNRELEASED
 
+**BREAKING CHANGES**
+
+-   Update matrix-rusk-sdk to `0.9.0`.
+-   Expose new API `DehydratedDevices.getDehydratedDeviceKey`, `DehydratedDevices.saveDehydratedDeviceKey`
+    and `DehydratedDevices.deleteDehydratedDeviceKey` to store/load the dehydrated device pickle key.
+    `DehydratedDevices.keysForUpload` and `DehydratedDevices.rehydrate` now use the `DehydratedDeviceKey` as parameter
+    instead of a raw UInt8Array.Use `DehydratedDeviceKey::createKeyFromArray` to migrate.
```

Comment:
Sorry, a belated review:

* I think this point about `keysForUpload` and `rehydrate` should be a separate bullet point
* The other points ("Update matrix-rusk-sdk to `0.9.0`" and "Expose new API") aren't breaking changes so should be *above* the "**BREAKING CHANGES**" header (see matrix-sdk-crypto-wasm v12.0.0, for example)
* It would be good to link to this pr for the changes (see lots of other entries in the changelog, for example)
* There are a number of typos on line 9. It should be: 
   ```
       instead of a raw `UInt8Array`. Use `DehydratedDeviceKey.createKeyFromArray` to migrate.
   ```